config-handler-router/src/test/groovy/org/codice/admin/router/ConfigurationHandlerRouterTest.groovy

```diff
@@ -0,0 +1,294 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.admin.router
+
+import org.codice.ddf.admin.api.config.Configuration
+import org.codice.ddf.admin.api.config.ConfigurationType
+import org.codice.ddf.admin.api.config.sources.WfsSourceConfiguration
+import org.codice.ddf.admin.api.handler.ConfigurationHandler
+import org.codice.ddf.admin.api.handler.report.CapabilitiesReport
+import org.codice.ddf.admin.api.handler.report.ProbeReport
+import org.codice.ddf.admin.api.handler.report.Report
+import spark.Request
+import spark.Response
+import spock.lang.Specification
+
+class ConfigurationHandlerRouterTest extends Specification {
+    
```

Comment:
All the tests also are setting at least one configuration handler, we should have cases where there are no configuration handlers set since this is likely the default state of the service.